.github/dependabot.yml

```diff
@@ -8,6 +8,11 @@ updates:
       - "dependencies"
       - "ruby"
       - "skip changelog"
+    groups:
```

Comment:
suggestion(non-blocking): I would remove this file as we prefer relying on upstream to manage dependencies

Reply:
Right! Fixed in https://github.com/Scalingo/python-buildpack/pull/94/commits/2c6caa94d2782bb464a128fb1352af6a9eb88667